client/src/pages/Home.js

```diff
@@ -0,0 +1,108 @@
+/*
+UI for home page.
+*/
+
+import React, { useContext, useRef } from "react";
+import { Button, makeStyles, Typography } from '@material-ui/core';
+import io from "socket.io-client";
+import { useHistory } from 'react-router-dom';
+//import { useUserState } from '../ContextProvider/user'; Will use these later
+//import { MatchContext } from '../ContextProvider/match'; this will be added 
```

Comment:
It's not great practice to include commented out code that will be used later. In this case I don't think it's worth removing but in the future try to avoid doing this. It can lead to a very cluttered dev branch and it wastes your time if the needs for a feature change before you uncomment